topics/dev/tutorials/tool-annotation/tutorial.md

```diff
@@ -0,0 +1,208 @@
+title: Annotation of a tool with EDAM ontology terms using bio.tools
+level: Introductory
+subtopic: tooldev
+questions:
+- How are Galaxy tools linked to EDAM ontology?
+- How to connect Galaxy tools to bio.tools?
+- What is bio.tools?
+- How to add, and update, entries in bio.tools?
+objectives:
+- Identify Galaxy tools without bio.tools entry
+- Create a bio.tools entry
+- Update a bio.tools entry 
+- Add EDAM ontology terms to a bio.tools entry
+- Link a Galaxy tool to its corresponding bio.tools entry
+time_estimation: 1H
+key_points:
+- Galaxy tools can get EDAM ontology terms from bio.tools
+- bio.tools entry can be created and modified to provide the best EDAM annotations
+- An up-to-date bio.tools entry provides readily accessible metadata that can help users find and understand tools that are available on Galaxy
+- bio.tools entry can easily be added to a Galaxy tool
+contributions:
+  authorship:
+    - bebatut
+    - supernord
+
+---
+
+Galaxy offers thousands of tools. Many of these tools either have incomplete metadata or are not yet linked to sources of high-quality metadata such as [bio.tools](https://bio.tools/).
+
+This prevents filtering for all tools in a specific research community or domain, and makes it all but impossible to employ advanced filtering with ontology terms like the ones from EDAM or to group tools based on an ontology to improve the Galaxy tool panel.
+
+[EDAM](https://edamontology.org/page) ({% cite black2021edam %}) is a comprehensive ontology of well-established, familiar concepts that are prevalent within bioscientific data analysis and data management. It includes 4 main sections of concepts (sub-ontologies):
+
+- **Topic**:  A category denoting a rather broad domain or field of interest, of study, application, work, data, or technology. Topics have no clearly defined borders between each other.
+- **Operation**> A function that processes a set of inputs and results in a set of outputs, or associates arguments (inputs) with values (outputs).
+- **Data**: Information, represented in an information artefact (data record) that is "understandable" by dedicated computational tools that can use the data as input or produce it as output.
+- **Format**: A defined way or layout of representing and structuring data in a computer file, blob, string, message, or elsewhere.
+
+![Simplified data flow diagram in EDAM architecture: boxes for concepts, lines for relations. Streamlined data management.](./images/EDAMrelations.png "EDAM architecture is simple. Boxes indicate top-level concepts (sections, sub-ontologies), and lines indicate types of relations. Source: <a href="https://edamontology.org/page">EDAM website</a>")
+
+The ontology can be navigated using [EDAM Browser](https://edamontology.github.io/edam-browser/):
+
+<iframe id="edam" src="https://edamontology.github.io/edam-browser/#operation_0291" frameBorder="0" width="80%" height="600px"> ![EDAM ontology browser](./images/edam_browser.png) </iframe>
```

Comment:
```suggestion
<iframe id="edam" src="https://edamontology.github.io/edam-browser/#operation_0291" frameBorder="1" width="80%" height="600px"> ![EDAM ontology browser](./images/edam_browser.png) </iframe>
```
I think it looks better with a boarder ... what do you think ?

Reply:
it's better for learners to recognise it is not a feature but some embedded resource, agreed.